src/content/docs/usage/installation.md

```diff
@@ -7,33 +7,34 @@ weight: 2
 ## Nextflow
 
 All nf-core pipelines use Nextflow, so this must be installed on the system where you launch your analysis.
-See [nextflow.io](https://www.nextflow.io/docs/latest/getstarted.html#installation) for the latest installation instructions. Once installed, see [_Nextflow configuration_](configuration.md) to set up Nextflow to run on your system.
+Once installed, see [_Nextflow configuration_](configuration.md) to set up Nextflow to run on your system.
+
+:::tip{collapse title="Official Nextflow docs"}
+If in doubt, please ee the [Nextflow installation docs](https://www.nextflow.io/docs/latest/getstarted.html#installation)
+for the latest instructions.
+Installation details are included here for convenience and may not be up to date.
+:::
 
 :::note
-You don't need to install the `nf-core` command line tools to run nf-core pipelines, you only need Nextflow. They can help with certain things though. Installation instructions for those are further down..
+You don't need to install the `nf-core` command line tools to run nf-core pipelines, you only need Nextflow.
+However, they offer a number of helpful functions and are essential for pipeline development using nf-core components.
+See the [tools page](/tools) for more information.
 :::
 
 ### Typical installation
```

Comment:
But you said it should already be installed and configured in the section above?

Reply:
I don't understand your comment here sorry.. Could you elaborate?